pandas/tests/io/excel/test_writers.py

```diff
@@ -293,7 +295,9 @@ def test_read_excel_parse_dates(self, tmp_excel):
         tm.assert_frame_equal(df2, res)
 
         res = pd.read_excel(tmp_excel, parse_dates=["date_strings"], index_col=0)
-        tm.assert_frame_equal(df, res)
+        expected = df[:]
+        expected["date_strings"] = expected["date_strings"].astype("M8[s]")
```

Comment:
Is this supposed to be using the `get_exp_unit` function that was also changed?

Reply:
no, that breaks several cases